Source/DafnyCore/Options/ShowSnippetsOption.cs

```diff
@@ -0,0 +1,20 @@
+namespace Microsoft.Dafny;
+
+/// <summary>
+/// This is an example of how to add an option that works for both the legacy and the new CLI UI.
+/// </summary>
+public class ShowSnippetsOption : BooleanOption {
+  public static readonly ShowSnippetsOption Instance = new();
+
+  public override string LongName => "showSnippets";
```

Comment:
AFAICT we standardized on hyphen-separated names (`--no-verify`, `--verification-time-limit`, etc), is there a reason this isn't `--show-snippets`?